QUEST_LV_0100.tsv

```diff
@@ -6403,101 +6403,101 @@ QUEST_LV_0100_20150317_006402	{memo X}Mr. Juan asked you to bring him Yukopus Le
-QUEST_LV_0100_20150323_006454	{memo X}$The evil force was first felt in Kvailas Forest. {nl}The force was very strong so it will remain this way for a while even if you defeat Bramble.
-QUEST_LV_0100_20150323_006455	{memo X}$The evil force of Kvailas Forest is becoming stronger and our followers are becoming weak. {nl}I hope you can help us. {nl}
-QUEST_LV_0100_20150323_006456	$Maybe the source of evil of Kvailas Forest has become the forest itself. {nl} I only hope this is just an idle fear.
-QUEST_LV_0100_20150323_006457	$Everything will be resolved when the evil force deep inside Kvailas Forest is dealt with. {nl}It's a pity everyone is after only what's right in front of them.
-QUEST_LV_0100_20150323_006458	Kvailas Forest is the worst. {nl}The evil force is at high and even your lives are at stake. 
-QUEST_LV_0100_20150323_006459	$I miss my family in Cobalt Forest. {nl}And.. I want to apologize to my brother for the fight we had.
-QUEST_LV_0100_20150323_006460	I remember singing and playing the the streams of Cobalt Forest
-QUEST_LV_0100_20150323_006461	Can you tell it to the elderly in Vieta Gorge? {nl}Just go a little upward to the left. 
+QUEST_LV_0100_20150323_006453	$When you have all materials ready, combine them at the Ershike Altar.
+QUEST_LV_0100_20150323_006454	{memo X}The evil force was first felt in Kvailas Forest. {nl}The force was very strong so it will remain this way for a while even if you defeat Bramble.
+QUEST_LV_0100_20150323_006455	{memo X}The evil force of Kvailas Forest is becoming stronger and our followers are becoming weak. {nl}I hope you can help us. {nl}
+QUEST_LV_0100_20150323_006456	$Perhaps the source of evil in Kvailas Forest has become the forest itself? {nl} I only hope this is just an ill-founded worry.
+QUEST_LV_0100_20150323_006457	$Everything will be resolved when the evil force deep inside Kvailas Forest is dealt with. {nl}It's a pity that everyone is only after what's right in front of them.
+QUEST_LV_0100_20150323_006458	$Kvailas Forest is the worst. {nl}The evil force is unquestionably concentrated and even your lives are at stake. 
+QUEST_LV_0100_20150323_006459	$I miss my family in Cobalt Forest. {nl}And... I want to apologize to my brother for the fight we had.
+QUEST_LV_0100_20150323_006460	$I remember singing and soaking my feet in the stream waters with my older brother in Cobalt Forest.
+QUEST_LV_0100_20150323_006461	$Can you inform this to the elderly in Vieta Gorge for me? {nl}If you veer left and move up  from here, you'll quickly reach her. 
 QUEST_LV_0100_20150323_006462	$The most important root must be somewhere in Kvailas Forest. {nl}Maybe we've been reacting around too much. {nl}Cutting off the head root will reduce the forces in this forest.
-QUEST_LV_0100_20150323_006463	I knew it was the poison of Evonipons. {nl}Its the only poison this strong. 
+QUEST_LV_0100_20150323_006463	$I knew it was Evoniphon's poison. {nl}There is no one else that will use a poison this strong. 
```

Comment:
"There is no one else that will use a poison this strong." -> "No one else would use a poison this strong."